packages/admin/src/Facades/FilamentNotificationManager.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Filament\Facades;
+
+use Filament\NotificationManager;
+use Livewire\Component;
+use Livewire\Response;
+use Illuminate\Support\Facades\Facade;
+
+/**
+ * @method static void notify(string $status, string $message)
+ * @method static Response handleLivewireResponses(Component $component, Response $response)
+ *
+ * @see NotificationManager
+ */
+class FilamentNotificationManager extends Facade
```

Comment:
The class is namespaced, no need for `Filament` in the class name.

Reply:
Fixed